bluebrain/repo-bluebrain/packages/sim-model/package.py

```diff
@@ -243,6 +243,9 @@ def _setup_run_environment_common(self, env):
             env.prepend_path("PYTHONPATH", self.prefix.lib.python)
         env.set("{}_ROOT".format(self.name.upper().replace("-", "_")), self.prefix)
 
+        if "+coreneuron" in self.spec:
+            env.set("CORENEURONLIB", self.prefix.lib + "/libcorenrnmech_ext.so")
+
         # ENV variables to enable Caliper with certain settings
```

Comment:
I believe you! new neuron master require this to be set? 🤔

Reply:
This is something that is more related to the removal of `CoreConfig.mod` and the latest changes in `NEURON` and how we find and load the `libcorenrnmech.*.so` library. Before the changes porposed in the mentioned Neurodamus PR, we were calling from the `CoreConfig.mod` a CoreNEURON function which existed in the `libcorenrnmech_ext.so`. During linking time of `libnrnmech.so` we were linking with the `CoreNEURON` library and everything worked. Now we don't use this `CoreNEURON` function but instead we use the Python API we don't link anything from `libcorenrnmech_ext.so` to `libnrnmech.so` so we have to dynamically load the CoreNEURON library during runtime with `CORENEURONLIB`